AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/Base Classes/ALHomeViewController.m

```diff
@@ -133,11 +138,14 @@ - (void)mailComposeController:(MFMailComposeViewController *)controller didFinis
     switch ( result )
     {
         case MFMailComposeResultSent:
-            [[[UIAlertView alloc] initWithTitle: @"Email Sent"
-                                        message: @"Thank you for your email, we will process it as soon as possible."
-                                       delegate: nil
-                              cancelButtonTitle: @"OK"
-                              otherButtonTitles: nil] show];
+        { UIAlertController* alert = [UIAlertController alertControllerWithTitle:@"Email Sent"
+                                                                         message:@"Thank you for your email, we will process it as soon as possible."
+                                                                  preferredStyle:UIAlertControllerStyleAlert];
+            
+            UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault
+                                                                  handler:^(UIAlertAction * action) {}];
+            
+            [alert addAction:defaultAction];}
```

Comment:
```
{
        UIAlertController* alert = [UIAlertController alertControllerWithTitle: @"Email Sent"
                                                                         message: @"Thank you for your email, we will process it as soon as possible."
                                                                  preferredStyle: UIAlertControllerStyleAlert];
            [alert addAction: [UIAlertAction actionWithTitle: @"OK" style: UIAlertActionStyleDefault handler: nil]];
}
```

Reply:
Since email functionality is not needed anymore, we are removing this part entirely